sdk/core/azure-core/src/main/java/com/azure/core/util/BinaryData.java

```diff
@@ -470,6 +470,35 @@ public static Mono<BinaryData> fromFlux(Flux<ByteBuffer> data, Long length, bool
         });
     }
 
+    /**
+     * Wraps the given flux in a BinaryData instance without subscribing to it.
+     *
+     * @param data Content to wrap.
+     * @param length Optional known length of the content in bytes.
+     * @return BinaryData instane wrapping the flux.
+     */
+    public static BinaryData wrapFlux(Flux<ByteBuffer> data, Long length) {
```

Comment:
Given that all other factory methods to create BinaryData starts with `from` prefix, this might not be easily discoverable with a different prefix. So, we should consider using `from` here as well - like `fromFluxSync()`?

Reply:
That is acceptable to me. I worry what someone may interpret from the phrasing `FluxSync` but if we think it is the better option then that's fine.